README.md

```diff
@@ -1,9 +1,176 @@
-# telestion-extension-template
+# telestion-extension-mavlink
 
-A template for creating new Telestion extensions.
+A Mavlink message parser for the Telestion backend.
 
-This template contains an initialized project structure with configured dependencies and a publishing pipeline.
+## Install
 
-To fetch the dependencies locally you need a Github personal access token with `packages:read` rights.
-Either add your github user name and the token as `GITHUB_ACTOR` and `GITHUB_TOKEN` to your environment variables 
-or fill the `.github.credentials.template`.
+First, add the extension package registry to your gradle repositories:
+
+```groovy
+repositories {
+    maven {
+        name = "GitHubMavlink"
+        url = uri("https://maven.pkg.github.com/wuespace/telestion-extension-mavlink/")
+        credentials {
+            username = System.getenv("GITHUB_ACTOR") == null ? GITHUB_ACTOR : System.getenv("GITHUB_ACTOR")
+            password = System.getenv("GITHUB_TOKEN") == null ? GITHUB_TOKEN : System.getenv("GITHUB_TOKEN")
+        }
+    }
+}
+```
+
+Next, add the extension to the project dependency list:
+
+```groovy
+dependencies {
+    // ...
+    implementation 'de.wuespace.telestion.extension:mavlink:0.4.0'
```

Comment:
```suggestion
    implementation 'de.wuespace.telestion.extension:mavlink:0.5.0'
```
Aren't we a little further ahead now? Let's use the newest version when merging ;)

Reply:
The latest version of this extension is currently `v0.3.7`. Should we jump two major versions? :thinking:

Reply:
Oh right, mixed that up with the core ^^
Sorry